src/components/molecules/VenueHero/VenueHero.tsx

```diff
@@ -1,49 +1,30 @@
 import React from "react";
 
-// Typings
-import { DetailsPreviewProps } from "./VenueHero.types";
-
-// Styles
 import * as S from "./VenueHero.styles";
-// import Button from "components/atoms/Button/Button";
-import { Link } from "react-router-dom";
-import { Button } from "react-bootstrap";
 
-const VenueHero: React.FC<DetailsPreviewProps> = ({
+export interface DetailsPreviewProps {
+  bannerImageUrl?: string;
+  logoImageUrl?: string;
+  name?: string;
+  subtitle?: string;
+  description?: string;
+  large?: boolean;
+}
+
+export const VenueHero: React.FC<DetailsPreviewProps> = ({
```

Comment:
`DetailsPreviewProps` is defined above, but yeah, should be renamed to `VenueHeroProps`

---

I think this may relate to the 'venue cards' being that get rendered for each venue we have access to? If so, perhaps `VenueCard` is a better name?

Heroes and cards in bootstrap for naming reference:

- https://getbootstrap.com/docs/5.0/examples/heroes/
- https://getbootstrap.com/docs/5.0/components/card/